src/libcollections/string.rs

```diff
@@ -433,6 +433,8 @@ impl String {
     ///
     /// [`str::from_utf8()`]: ../../std/str/fn.from_utf8.html
     ///
+    /// The inverse of this method is [`as_bytes`](#method.as_bytes).
```

Comment:
Please don't use this link syntax. Move the url below and just keep `[as_bytes]` here.